webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -234,12 +236,20 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
             return;
         }
 
-        const updatedPost = {
+        let updatedPost = {
             message: editText,
             id: editingPost.postId,
             channel_id: editingPost.post.channel_id,
         };
 
+        const hookResult = await actions.runMessageWillBeUpdatedHooks(updatedPost, editingPost.post);
+        if (hookResult.error) {
+            setPostError((<>{hookResult.error.message}</>));
```

Comment:
Nit: one of these parens is redundant:

```suggestion
            setPostError(<>{hookResult.error.message}</>);
```

Reply:
Yeah, I think the simple check should be enough though as it would make an empty string semantically equivalent to no error being present. Adding a default message feels a bit too much involved to cover the edge case of a broken/misimplemented plugin.

Reply:
Removed the extra parenthesis and added a simple check before calling setPostError